.github/workflows/ci.yml

```diff
@@ -58,6 +58,18 @@ jobs:
       - name: test
         run: make test TEST_ARGS="-vv --showlocals"
 
+      - name: test (timestamp-authority)
```

Comment:
FLAG: This tests the new Timestamping Functionalities by downloading `sigstore/timestamp-authority`.

This could either use an external (and reliable) TSA or be more aggressively cached.